spyder/plugins/configdialog.py

```diff
@@ -317,7 +317,12 @@ def load_from_conf(self):
             radiobutton.toggled.connect(lambda _foo, opt=option:
                                         self.has_been_modified(opt))
         for lineedit, (option, default) in list(self.lineedits.items()):
-            lineedit.setText(self.get_option(option, default))
+            option_value = self.get_option(option, default)
+            if isinstance(option_value, tuple):

```

Comment:
Why not use lists or tuples for both? I find this very confusing, so I really want you to decide for one or the other :-)